README.md

```diff
@@ -120,3 +120,47 @@ npm run docusaurus docs:version 0.4.5
 ```
 
 If you are developing in Solidity and Foundry you can use `[forge-doc](https://book.getfoundry.sh/reference/forge/forge-doc)`
+
+### Custom Elements
+
+#### Detail Element
+
+This is a Detail element that contains other text inside. If you format the summary section as shown it renders markdown correctly.
+
+```
+<details className="customDetails">
+
+  <summary>
+  
+  ## Title Here
+  
+  </summary>
+
+### Subtitles as needed
+
+content here
+
+</details>
+```
+
+There is also a "customFaqDetails" css class that removes the borders.
+
+#### PrettyLink
+
+The PrettyLink element makes your links into button-like elements with subtle animation and yearn styling. These links will fill the full width of the markdown document. Can be used with naked links or with markdown style links.
```

Comment:
cant this be just <Link> ? or is Link being used already? normally the web standard would be like

`<Link>` for the "non pretty" variant
and
`<Link variant="pretty">` for the pretty variant

Reply:
Going to merge this PR and then we can discuss whether this is the right way to do the PrettyLink. Will change it if there is a better way.